content/en/docs/setup/install/external-controlplane/index.md

```diff
@@ -424,6 +424,9 @@ including gateways, if needed.
 
 1. Enable an ingress gateway on the remote cluster:
```

Comment:
The numbering isn't quite correct. We have 3 number 1's at the top of the list. I'm not sure if indentation of the tabsets will make things correct or not.

Reply:
@ericvn That worked. Made the changes and it looks good. Thanks for the help. Appreciate it.